interop/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
```

Comment:
FYI you have not declared this new module in the sessings.gradle in corda-api - so as far as gradle is concerned it will know nothing about this module.

you need to specify this in the included submodules here

https://github.com/corda/corda-api/blob/release/os/5.1/settings.gradle#L87-L104

your runtime-os equivalent branch should also update it's composite build logic to include this, like so 

https://github.com/corda/corda-runtime-os/blob/release/os/5.1/settings.gradle#L458-L476

Reply:
Thanks @ronanbrowne